wipac_dev_tools/__init__.py

```diff
@@ -14,6 +14,7 @@
     "argparse_tools",
     "data_safety_tools",
     "timing_tools",
+    "prometheus"
```

Comment:
Do you mind calling it `prometheus_tools`? I adopted the pattern so that on imports there would be no naming conflicts, ie `from wipac_dev_tools import prometheus_tools; import prometheus`.

Reply:
Can do.